tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -21,16 +21,57 @@
 #include "llvm/Support/Path.h"
 #include "llvm/Support/raw_ostream.h"
 
+namespace {
+spv::ExecutionModel getSpirvShaderKindFromHlslProfile(const char *profile) {
+  // DXIL Models are:
+  // Profile (DXIL Model) : HLSL Shader Kind : SPIRV Shader Kind
```

Comment:
We should prefer to use `SPIR-V` in non-code scenarios. `SPIR-V` is the official spelling; we use `SPIRV` b/c `-` is not welcome in code identifiers.